pkg/util/ipset/ipset.go

```diff
@@ -322,4 +322,20 @@ func validatePortRange(portRange string) bool {
 	return true
 }
 
+// IsNotFoundError returns true if the error indicates "not found".  It parses
+// the error string looking for known values, which is imperfect but works in
+// practice.
+func IsNotFoundError(err error) bool {
+	es := err.Error()
+	if strings.Contains(es, "does not exist") {
```

Comment:
is there a way we can write a test to detect if these strings change?

Reply:
> is there a way we can write a test to detect if these strings change?

I am not sure fully understand your meaning. Do you mean write a test to wath the ipset message string?

These strings are very stable and have not changed for many years.